nuxeo-runtime/nuxeo-stream/src/main/java/org/nuxeo/lib/stream/log/kafka/KafkaLogAppender.java

```diff
@@ -67,14 +65,27 @@
 
     protected final KafkaNamespace ns;
 
+    protected final Codec<M> codec;
+
     protected KafkaProducer<String, Bytes> producer;
 
     protected boolean closed;
 
+<<<<<<< fb1d9d00148818be69325582ac32ac7ece826c64
```

Comment:
These kinds of errors should be fixed by amending this commit, not by adding a further commit to fix it. Here I'm reviewing commit by commit and I don't know what the code is supposed to look like.